CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.5.0
+
+**New Features**
+
+- Particle migration using Cajita grid added
+- Random particle generation added
+- Complete Cajita tutorial examples added
+- Cajita performance benchmarks added
+- Cabana Docker container added
```

Comment:
Not sure what this refers to

Reply:
Maybe confusing to include - the container Christoph added with Github Actions for convenience

Reply:
Removed